src/orders/components/OrderPayment/styles.ts

```diff
@@ -36,6 +36,11 @@ export const useStyles = makeStyles(
     success: {
       color: theme.palette.success.dark,
     },
+    actions: {
+      "& > *:not(:last-child)": {
+        marginRight: theme.spacing(2),
+      },
+    },
```

Comment:
Wouldn't be the following cleaner?
```suggestion
    actions: {
      gap: theme.spacing(2),
      display: "flex",
      flexWrap: "wrap",
      justifyContent: "right"
    },
```
Gap for flex is not only supported by IE 11, but it is by Edge.

Reply:
why is it cleaner? :D

Reply:
The purpose of `gap` is clear, we want to create a gap between items, while when writing `marginRight` we require from dev to understand it's purpose. With `gap` browser takes care about not appending space at the very end, so if we one day decide to add specific margin to one item, it will not collide with "& > *:not(:last-child)" rule, that rule is implicitly in `gap` property.

Besides that with `flexWrap: "wrap"` we also apply gap between rows which may wrap on small resolutions and justify them to right with `justifyContent: "right"`. Otherwise, we should need to write another margin rule.